src/components/Auth/RegisterCheckBox.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useCallback } from 'react';
+import * as S from 'components/Auth/styles';
+import styled from 'styled-components';
+
+const CheckDotLabel = styled.label`
+  display: inline-block;
+  vertical-align: middle;
+  align-items: center;
+  cursor: pointer;
+`;
+
+const CheckDot = styled.input.attrs({ type: 'checkbox' })`
+  display: none;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+`;
+
+const CustomCheckDot = styled.div`
+  display: inline-block;
+  width: 13px;
+  height: 13px;
+  border-radius: 50%;
+  margin-right: 20px;
+  border: solid 1px #c4c4c4;
+  transition: all 150ms;
+  ${CheckDot}:checked + & {
+    border: solid 1px #ffd25d;
+    background-color: #ffd25d;
```

Comment:
styled-components 는 페이지 컨셉 색상들을 `theme` 를 사용해서 관리할 수 있습니다.
필수는 아니지만, 프로젝트를 사용할 때 공통적인 속성 공유를 위해 자주 사용하는 방법이니 관심있으면 수정해보는 것도 좋을듯 합니다. (한번 세팅해놓으면 사용할 때 뿌듯하답니다 :) )

```suggestion
    border: solid 1px ${theme.borderColor};
    background-color: ${theme.borderColor};
```